web/public/locales/en/common.json

```diff
@@ -508,32 +508,32 @@
+    "communityDescription": "The Helpbuttons community is the backbone of this project, providing it with sustainability. This tool is designed for adaptability to a multitude of purposes, thereby benefitting a diverse range of users while also encouraging support for the project.\n\nBy federating with other Helpbuttons apps, you can share users and resources.This method of collaboration enables us to unite more people and ultimately improve the tool we provide.\n\nYou can find the source code of this app at <a href=\"https://helpbuttons.org\">Helpbuttons.org</a>",
+    "ethicsDescription": "At Helpbuttons, we're committed to maintaining the highest standards of ethical conduct. We believe in transparency, honesty, and respect for all individuals. \n\nOur software is designed to connect people and facilitate sharing, which is a responsibility we take seriously. We encourage our users to use our platform with the same values in mind, prioritizing respectful communication and mindful sharing. Any misuse of the software, including exploitation, harm to others, or manipulation of the platform's functionalities for personal gain, will not be tolerated. Users are required to adhere to these ethics policies at all times.\n\nIMPORTANT: Please take into account this is an open software and we cannot be responsible of the misuse of this software outside of our servers. If you find a misuse of this software, please contact us immediately.",
+    "privacyDescription": "If you are a regular user:\n\nAdministrators of this network are responsible for your data in first instance. Check who are they in the Home Page of this app. If their privacy info is not available in this page, we recommend to ask them where are they storing your data and how do they manage it. If they're using our servers, our privacy policy applies.\n\n If you are an administrator:\n\nBy being an administrator you become responsible of the use of the data and the software.\n\nThe legal responsible for Helpbuttons software and servers is Helpbuttons - Watch Out Freedom G67992032, with address Avda. Mexico 10, 11405 Jerez, Cádiz, Spain. At Helpbuttons, we respect your privacy and are committed to protecting your personal data. \n\nIf you use our servers, our privacy policy outlines how we collect, store, use, and disclose your personal information. We comply with all relevant privacy laws and regulations, ensuring your data is used solely for the purpose of providing and improving our service. \n\nWe are transparent about our data collection processes, we never collect data with commercial purposes and you have the right to access, update, or delete your personal data at any time. \n\nRemember, sharing personal information in this app is at your discretion—be mindful and only share what you are comfortable with others knowing.",
```

Comment:
this needs to be configured on the network not hard coded in the F.A.Q. 
and you can't just be changing the terms when you have so many instances running already.

Reply:
this is the prinicples we've been using from the beggining but phrased more evident

Reply:
how should we proceed in that case?